draft-ietf-rtcweb-jsep.xml

```diff
@@ -1126,9 +1128,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t hangText="require:">The browser will only
                         gather RTP candidates. This halves the number
                         of candidates that the offerer needs to gather.
-                        When acting as answerer, the browser will
-                        reject any m= section that does not provide an
-                        "a=rtcp-mux" attribute.</t>
+                        When acting as answerer, the implementation
+                        will reject any m= section that does not
+                        contain an "a=rtcp-mux" attribute.</t>
```

Comment:
prefer that it not reject it but instead just treat it as if it does "a=rtcp-mux"